addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -516,7 +519,10 @@ var KanbanRenderer = BasicRenderer.extend({
         if (this.state.groupedBy.length) {
             _.invoke(this.widgets, 'cancelQuickCreate');
         }
-        this._toggleNoContentHelper();
+        // if quick create adds a record then do not show no content helper
+        if (!this.$('.o_kanban_record').length) {
```

Comment:
this doesn't seem right.
We should rely on this.state, just as the method `_toggleNoContentHelper` when it calls `_hasContent` (which juste does, in basicRenderer `return this.state.count !== 0`)
I checked, In 12.0, when coming back from the create after validating the quickCreate, the group's count (ie the column's count) is updated, but soon after; like right after `createRecordInGroup`, count is 0 again, and res_ids has been emptied.
There is some weird shit going on here.

Reply:
@kebeclibre I need to recall why I did fix based on DOM while yes you are right we should rely on ```state.count``` I will check once again soon